scripts/utils/price-utils.js

```diff
@@ -139,9 +139,52 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return true
   }
 
+  const checkNoProfitableOffer = async (order, exchange, currentMarketPrice) => {
+    const tokenInfo = fetchTokenInfoFromExchange(exchange, [order.buyToken, order.sellToken])
+
+    if (currentMarketPrice === undefined) {
+      currentMarketPrice = await getDexagPrice(
+        (await tokenInfo[order.buyToken]).symbol,
+        (await tokenInfo[order.sellToken]).symbol
+      )
+    }
+    // checks whether the order amount is negligible
+    if ((await valueInUSD(order, tokenInfo)) < 10) {
+      return true
+    }
+
+    // multiplicator is used to counter rounding errors
+    const multiplicator = 1000000000
+    const marketPrice = new BN(currentMarketPrice * multiplicator)
```

Comment:
If `currentMarketPrice * multiplicator` is not an integer, then this line fails. This might be an issue for prices like `0.33333333333333333`. Also, we implicitly assume currentMarketPrice > 10^-9, are we ok with this?

Reply:
I just tested it again: `const t = new BN(1.34)` gives the same as new BN(1).

To be honest, I don't have a good solution for it right now. because if `currentMarketPrice * multiplicator` gets too big, we get a scientific representation, as eg. 4.546e30 and then this throws as well.

Reply:
Actually, I found a good way :) Thank pushing my focus to the rounding issues again